beast/tests/test_regresscheck.py

```diff
@@ -326,24 +350,26 @@ def test_fit_grid(self):
 
     # ###################################################################
     # AST tests
-    @pytest.mark.skip(reason="updated cached file needed")
+    @pytest.mark.skip(reason="need filters info: get from sed grid?")
```

Comment:
Could you clarify what's needed here?  I'm surprised it's not working.

Reply:
Like all the tests - depends on if it is phat_small or metal_small set of cached files.  This test would work with either, but needs to know the filters.   The filters change between the two cached file sets.  This is a note to indicate this (not a great note - but at least a clue).  I figured a separate PR (or multiple PRs) to get the regression tests all working *and* setup for being able to have different cached sets.  This will be very useful in the future when we break backwards compatability.

Reply:
I've added a bit more info to make this comment more helpful.